src/java/org/apache/cassandra/db/compaction/CompactionStrategyManager.java

```diff
@@ -177,7 +177,7 @@ public int getIndexForSSTableDirectory(Descriptor descriptor)
         holders = ImmutableList.of(transientRepairs, pendingRepairs, repaired, unrepaired);
 
         cfs.getTracker().subscribe(this);
-        logger.trace("{} subscribed to the data tracker.", this);
+        logger.trace("Compaction manager for {}.{} subscribed to the data tracker.", cfs.keyspace.getName(), cfs.name);
```

Comment:
"this" is called on an object which does not have toString overriden so I doubt that the current log message is actually meaningul.